internal/backend/crypto/gpg/cli/binary_others.go

```diff
@@ -4,11 +4,24 @@ package cli
 
 import (
 	"os/exec"
+
+	"github.com/gopasspw/gopass/pkg/debug"
+	"github.com/gopasspw/gopass/pkg/pinentry/gpgconf"
 )
 
 func detectBinary(name string) (string, error) {
-	if name == "" {
-		name = "gpg"
+	// user supplied binaries take precedence
+	if name != "" {
+		return exec.LookPath(name)
+	}
+	// try to get the proper binary from gpgconf(1)
+	if p, err := gpgconf.Path("gpg"); err == nil && p != "" {
+		debug.Log("gpgconf returned %q for gpg", p)
+		return p, nil
+	} else {
```

Comment:
This will make our CI complain that there is a else statement before a return statement and thus it could be dropped.